lib/matplotlib/tests/test_figure.py

```diff
@@ -838,7 +838,7 @@ def test_reused_gridspec():
                   savefig_kwarg={'facecolor': 'teal'},
                   remove_text=False)
 def test_subfigure():
-    np.random.seed(19680808)
+    np.random.seed(19680801)
```

Comment:
Were the images being updated already? Maybe no need to change this one otherwise.

Reply:
Ooops, good point.  Like they aren't in a release yet, but yes they are in the repo...

Reply:
OK< so they did change.  Even if I kept the old (incorrect) seed, there was a slight horizontal shift, I assume due to the slot in place for the supylabel.

Reply:
OK, seems fine to change then.